CHANGELOG.md

```diff
@@ -1,237 +1,287 @@
-CHANGELOG
-=========
+# CHANGELOG
 
 ## HEAD (Unreleased)
+
 _(none)_
 
 ---
 
+## 3.5.1 (2020-05-12)
+
+- Add retries on host key retrieval for Function Apps
+
 ## 3.5.0 (2020-05-12)
-* Upgrade to pulumi-terraform-bridge v2.3.1
-* Fix `error404document` mapping on `azure.storage.Account`
-* Upgrade to v2.9.0 of the AzureRM Terraform Provider
-* Update ArchiveFunctionApp to pass correct params to FunctionApp
+
```

Comment:
Could you please undo all the changes to the changelog except for your new entry?

Reply:
Rectified - apologies for this